tests/build-remote.sh

```diff
@@ -5,26 +5,29 @@ clearStore
 if ! canUseSandbox; then exit; fi
 if ! [[ $busybox =~ busybox ]]; then exit; fi
 
-chmod -R u+w $TEST_ROOT/machine0 || true
-chmod -R u+w $TEST_ROOT/machine1 || true
-chmod -R u+w $TEST_ROOT/machine2 || true
-rm -rf $TEST_ROOT/machine0 $TEST_ROOT/machine1 $TEST_ROOT/machine2
-rm -f $TEST_ROOT/result
-
 unset NIX_STORE_DIR
 unset NIX_STATE_DIR
 
+function join_by { local d=$1; shift; echo -n "$1"; shift; printf "%s" "${@/#/$d}"; }
+
+builders=(
+  "ssh://localhost?remote-store=$TEST_ROOT/machine1"
+  "$TEST_ROOT/machine2 - - 1 1 foo"
+  "ssh-ng://localhost?remote-store=$TEST_ROOT/machine3?systemFeatures=baz - - 1 1 baz"
```

Comment:
`?systemFeatures=baz` won't work, but it's failing before it even gets that far.